src/patternfly/components/Button/button.scss

```
@@ -279,6 +279,8 @@
   --#{$button}__icon--MarginInlineEnd: 0;
   --#{$button}__icon--m-start--MarginInlineEnd: 0;
   --#{$button}__icon--m-end--MarginInlineStart: 0;
+  --#{$button}__icon--AnimationDuration--notify: var(--pf-t--global--duration--600);
+  --#{$button}__icon--AnimationTimingFunction--notify: var(--pf-t--global--motion--timing-function--default);
```

Technically, this is how we would normally write out this var. I'm cool leaving it the way you have it, since we have an identifier as `--notify` on the end. WDYT? 

```suggestion
  --#{$button}--m-notify__icon--AnimationDuration--notify: var(--pf-t--global--duration--600);
  --#{$button}--m-notify__icon--AnimationTimingFunction--notify: var(--pf-t--global--motion--timing-function--default);
```